### PR TITLE
Add changelog and the updated pull request template

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

## Remarks:
https://github.com/ballerina-platform/ballerina-standard-library/issues/1198
https://github.com/ballerina-platform/ballerina-standard-library/issues/1197

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+Fixes:
```
Comment (shafreenAnfar) [inline .github/pull_request_template.md:2]: Lets remove this.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (shafreenAnfar) [inline .github/pull_request_template.md:1]: Add ## Purpose

File: changelog.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina RabbitMQ package through the releases.
```
Comment (shafreenAnfar) [inline changelog.md:2]: Follow this

https://github.com/ballerina-platform/module-ballerina-websocket/pull/113/files#diff-3bd14d078188074c410028847113ceae68865d0ad5b844a27183ef87fbe2fcc3

Comment (aashikam) [reply changelog.md:2]: Fixed in ca9c9e65dcd85a834e5fbb7f51d7d6aef3f82936. And nothing to mention in other sections (i.e Added, Removed, Fixed etc)

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**